pkg/api/testing/fuzzer.go

```diff
@@ -252,8 +252,9 @@ func FuzzerFor(t *testing.T, version string, src rand.Source) *fuzz.Fuzzer {
 			s.Phase = api.NamespaceActive
 		},
 		func(http *api.HTTPGetAction, c fuzz.Continue) {
-			c.FuzzNoCustom(http)        // fuzz self without calling this function again
-			http.Path = "/" + http.Path // can't be blank
+			c.FuzzNoCustom(http)                        // fuzz self without calling this function again
+			http.Path = "/" + http.Path                 // can't be blank
+			http.Scheme = api.URIScheme(c.RandString()) // can't be blank
```

Comment:
@thockin @mikedanese I am confused here: I added a field with a default, and updated the fuzzer here to ensure it is set to non-empty strings so that the default does not kick in. However, I am getting errors like this:

```
--- FAIL: TestRoundTripTypes (0.66s)
	serialization_test.go:70: 1: Pod: diff: (*api.Pod){...HTTPGet:(*api.HTTPGetAction){...Scheme:(api.URIScheme)
		A: } 
		B: http} 
```

This means that the value is getting fuzzed to `""` initially and re-set by the defaulter, but why is this happening?